src/main/resources/org/jenkinsci/plugins/pipeline/utility/steps/conf/ReadPropertiesStep/help.html

```diff
@@ -0,0 +1,49 @@
+<!--
+  ~ The MIT License (MIT)
+  ~
+  ~ Copyright (c) 2016 CloudBees Inc.
+  ~
+  ~ Permission is hereby granted, free of charge, to any person obtaining a copy
+  ~ of this software and associated documentation files (the "Software"), to deal
+  ~ in the Software without restriction, including without limitation the rights
+  ~ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  ~ copies of the Software, and to permit persons to whom the Software is
+  ~ furnished to do so, subject to the following conditions:
+  ~
+  ~ The above copyright notice and this permission notice shall be included in all
+  ~ copies or substantial portions of the Software.
+  ~
+  ~ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  ~ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  ~ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  ~ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+  ~ SOFTWARE.
+  -->
+<p>
+    Reads a file in the current working directory or a String as a plain text
+    <a href="https://docs.oracle.com/javase/7/docs/api/java/util/Properties.html" target="_blank">Java Properties</a>
+    file.
+    The returned object is a normal Map with String keys.
+    The map can also be pre loaded with default values before reading/parsing the data.
+</p>
+<strong>Fields:</strong>
+<ul>
+    <li><code>file</code>: Optional path to a file in the workspace to read the properties from.</li>
+    <li><code>text</code>: An Optional String containing the properties format.</li>
+    <li><code>defaults</code>: A Map containing default key/values.</li>
+</ul>
+<p>
+    <strong>Example:</strong><br/>
+    <code>
+        <pre>
+        def d = [test: 'Default', something: 'Default', other: 'Default']
+        def props = readProperties defaults: d, file: 'dir/my.properties', text: 'other=Override'
+        assert props['test'] == 'One'
+        assert props['something'] == 'Default'
+        assert props.something == 'Default'
+        assert props.other == 'Override'
+        </pre>
```

Comment:
`def d = [test: 'Default', something: 'Default', other: 'Default']` should be
`def d = [test: 'One', something: 'Default', other: 'Override']` then? Or am I missing something?